README.md

```diff
@@ -1,83 +1,99 @@
 # EmojiConverter
-This is an Android library used to convert text to Emojis in edittext. This is the most stable version of this library till date. This is under more modifications. Use It For Free.
+This is a HMOS library used to convert text to Emoji's from a textfield or from other sources.  
 
-[![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-Emoji--Converter-brightgreen.svg?style=flat)](https://android-arsenal.com/details/1/6534)
 
-## Preview
+## Source
+The code in this repository was inspired from [AnkitKiet/EmojiConverter - Release 1.15](https://github.com/AnkitKiet/EmojiConverter). 
+We are very thankful to AnkitKiet. 
 
-![Sample](/ezgif-4-63891ff6bd.gif?raw=true "Preview")
+## Feature
+This library converts text to it's corresponding emoji .
 
-## Installation
-### Gradle
-Add below code in build.gradle
+![finaldemo](https://user-images.githubusercontent.com/60657039/129444769-961ac72c-85b2-41b5-b45d-aadd904349ce.gif)
+
+## Dependency
+1. For using EmojiConverter module in sample app, include the source code and add the below dependencies in entry/build.gradle to generate hap/support.har.
 ```
-allprojects {
-	repositories {
-		maven { url 'https://jitpack.io' }
-		}
+	dependencies {
+		implementation project(':emojiconverterlibrary')
+        	implementation fileTree(dir: 'libs', include: ['*.har'])
+        	testImplementation 'junit:junit:4.13'
 	}
 ```
-Add a dependency
+2. For using EmojiConverter in separate application using har file, add the har file in the entry/libs folder and add the dependencies in entry/build.gradle file.
 ```
-dependencies {
-	compile 'com.github.AnkitKiet:EmojiConverter:1.14'
+	dependencies {
+		implementation fileTree(dir: 'libs', include: ['*.har'])
+		testImplementation 'junit:junit:4.13'
 	}
-```
 
-### Maven
-Add below code for Maven installation
-```
-<repositories>
-		<repository>
-		    <id>jitpack.io</id>
-		    <url>https://jitpack.io</url>
-		</repository>
-	</repositories>
 ```
-Add a dependency
+3. For using MaterialLetterIcon from a remote repository in separate application, add the below dependencies in entry/build.gradle file.
```

Comment:
This needs to be removed. Must be added after Maven release

Reply:
removed that part